src/workerd/api/crypto/keys.c++

```diff
@@ -90,7 +90,7 @@ kj::Array<kj::byte> AsymmetricKeyCryptoKeyImpl::exportKeyExt(
   auto bio = OSSL_BIO_MEM();
 
   struct EncDetail {
-    char* pass = &EMPTY_PASSPHRASE[0];
+    char* pass = const_cast<char*>(&EMPTY_PASSPHRASE[0]);
```

Comment:
I haven't used const_cast before – let me know if this usage pattern is incorrect.

Reply:
I think this case is ok since we don't expect anything to be written.